simvue/client.py

```diff
@@ -80,6 +81,33 @@ def __exit__(self, type, value, tb):
             self.set_status('completed')
             self._send_metrics()
 
+    def _heartbeat(self):
+        """
+        Send heartbeats at regular intervals
+        """
+        start = time.time()
+        while not self._event.is_set():
+            time.sleep(1)
+            if time.time() - start > 60:
```

Comment:
Wondering if it will be easier to have a separate config class for these parameters as it may need to be tweaked?

Reply:
I suppose that could be useful to have, as there are other things that have parameters (or implicit parameters), such as HTTP timeouts, e.g. interacting with both the Simvue REST API and also object storage API.

Reply:
How about something like what's recommended here in the answer with 36 likes (i.e. the second one)? https://stackoverflow.com/questions/6198372/most-pythonic-way-to-provide-global-configuration-variables-in-config-py

Reply:
Looks good. I was thinking in terms of logging as well and perhaps having a base class from which we can derive to store data for different cases, so was thinking of the @dataclass decorator? https://realpython.com/python-data-classes/

Reply:
Do you mean like htis? https://github.com/simvue-io/client/commit/b584e7ddfe446b6ae3bab25cc38054f52fa34902